partials/tabs/coalaonline.html

```diff
@@ -274,3 +274,9 @@ <h6 class="uppercase"><strong><span class="lowercase">c</span>OAFILE</strong></h
     </div>
   </div>
 </div>
+<script>
+$(document).ready(function(){
+  $('ul.tabs').tabs();
+});
+
+</script>
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/coalaonline.html
+++ b/tmp/tmp8v872ato/partials/tabs/coalaonline.html
@@ -279,4 +279,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```